web-app/react-app/components/AuthPage/schemas/common.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import * as yup from 'yup'
+import zxcvbn from 'zxcvbn'
+
+export const email = yup.string()
+    .trim()
+    .required('Email is required')
+    .email('Email must be a valid email address')
```

Comment:
This wording should probably match the backend's response on email validation `Please enter a valid email address`